graph/api/src/main/java/org/jboss/windup/graph/model/BelongsToProject.java

```diff
@@ -5,13 +5,6 @@
  */
 public interface BelongsToProject
 {
-    /**
```

Comment:
This change is unrelated, but I removed it as the operation was unused and inherently unsafe. It doesn't deal with shared libraries correctly.

Reply:
It is actually not unused, it is used here: https://github.com/windup/windup/blob/master/rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/stats/TechnologiesStatsService.java#L259

Reply:
I get the previous comment as "It is actually used,..." :)

But the referenced  TechnologyStatsService usage is not using the removed method.

I find it https://github.com/windup/windup-web/blob/master/addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/rest/graph/EJBResourceImpl.java#L68 which now show the failure in http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-web-nightly/687/